pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -80,16 +75,7 @@ func getDBImages(extensionState testhelpers.ExtensionState) (prev string, clean
 		//but migration code that works in an older PG version should generally work in a newer one.
 		panic("Only use pg12 for upgrade tests")
 	}
-	switch {
-	case extensionState.UsesMultinode():
-		return "timescaledev/promscale-extension:0.1.1-ts2-pg12", testhelpers.LatestDBWithPromscaleImageBase + ":latest-ts2-pg12"
-	case !extensionState.UsesTimescaleDB():
-		return "timescale/timescaledb:latest-pg12", "timescale/timescaledb:latest-pg12"
-	case extensionState.UsesTimescale2():
-		return "timescaledev/promscale-extension:0.1.1-ts2-pg12", testhelpers.LatestDBWithPromscaleImageBase + ":latest-ts2-pg12"
-	default:
-		return "timescaledev/promscale-extension:0.1.1-ts1-pg12", testhelpers.LatestDBWithPromscaleImageBase + ":latest-ts1-pg12"
-	}
+	return "timescaledev/promscale-extension:0.1.1-ts2-pg12", "cevian/promscale-extension" + ":latest-ts2-pg12"
```

Comment:
replace `cevian` with `timescaledev`?

Reply:
This is a temporary fix while our extension is re-released (see commit message)

Reply:
Now removed